diesel/src/result.rs

```diff
@@ -188,6 +188,12 @@ pub trait DatabaseErrorInformation {
     /// An optional integer indicating an error cursor position as an index into
     /// the original statement string.
     fn statement_position(&self) -> Option<i32>;
+
+    /// An error code indicating the cause of the error.
+    ///
+    /// Currently this method will return `None` for all backends other than
+    /// PostgreSQL.
```

Comment:
maybe we want to add a link to here:
https://www.postgresql.org/docs/current/errcodes-appendix.html

to the doc?